src/hooks/useHotelsSearch.js

```diff
@@ -0,0 +1,16 @@
+import { useDispatch, useSelector } from "react-redux";
+import { searchHotels } from "../store/actions/hotelsSearch.action";
+
+const useHotelSearch = () => {
+  const dispatch = useDispatch();
+  const foundHotels = useSelector((state) => state.hotels.state);
+  const loading = useSelector((state) => state.hotels.loading);
```

Comment:
Pay attention, you can combine such selectors. It can help to call useSelector just once.
```suggestion
  const { foundHotels, loading } = useSelector(({ hotels }) => ({
    foundHotels: hotels.state,
    loading: hotels.loading
  }));
```